### ISSUE TITLE
Error on iojs 3.0.0

### ISSUE THREAD

**Issue by dukex** [aexmachina/ember-cli-sass#66]
The node-sass don't support iojs 3.0.0, I tried to run my ember-cli project on new iojs version and receive a error, to reproduce: 
```
nvm use iojs // 3.0.0
npm install ember-cli-sass // will broken when npm try to compile node-sass
```

eg: https://travis-ci.org/uhuraapp/uhura-frontend/jobs/76277392

The problem is on node-sass, was release node-sass 3.3.0 to fix this problems, see the issue
 https://github.com/sass/node-sass/issues/1053

**Comment by dukex** [comment]
The 4.1.0 was release just now with node-sass update to 3.2.0 :disappointed: 
I should have created this issue before, but I waited node-sass be released, sorry guys

**Comment by aexmachina** [comment]
Looks like this will be fixed in 3.3.0, which hasn't been released yet. When it's released let me know and I'll publish an update.

**Comment by dukex** [comment]
Hi @aexmachina, node-sass 3.3.1 was released, The 3.3.0 had a problem with fsevents, fixed in 3.3.1

**Comment by aexmachina** [comment]
Thanks, I'll update in the morning

**Comment by aexmachina** [comment]
Published 4.2.0

**Comment by dukex** [comment]
<3 Thank you!
